cmd/cstor-volume-grpc/api/handler.go

```diff
@@ -0,0 +1,65 @@
+package api
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/openebs/maya/pkg/util"
+	"golang.org/x/net/context"
+)
+
+// sudo $ISTGTCONTROL snapdestroy vol1 snapname1 0
+// sudo $ISTGTCONTROL snapcreate vol1 snapname1
+
+// constants
+const (
+	VolumeGrpcListenPort = 7777
```

Comment:
let's make this configurable but default to 7777?

Reply:
made it configurable through command line parameter --port for the server command start